### DISCUSSION TITLE
How to Make Side Menu Collapsible

### DISCUSSION THREAD

**Discussion by AdamFermier** [McShelby/hugo-theme-relearn#583] [Q&A] [answered]
Hi Folks - When I view the theme on my phone it collapses the menu to a hamburger menu in top left side - great - love it!  How might I do that when viewing on my machine to get wider display area (enable hide menu into hamburger menu if user chooses to collapse it).  

Thanks!  Adam

**Comment by McShelby** [comment, marked as answer]
There's no build in way and I hesitate to support this.

Reason is, that the content width is locked to a certain max size anyways. Therefore it will not pass past this limit to assert readability.

You could modify the CSS to overcome those mechanism but this would not be future proof as the DOM and therefore the CSS selectors are not guaranteed to be stable between even minor releases.
